### ISSUE TITLE
"apb relist" fails to trigger service-catalog relist with "404 page not found"

### ISSUE THREAD

**Issue by djwhatle** [fusor/ansible-playbook-bundle#144]
Using the latest service-catalog, latest origin, and canary ansible-service-broker, I'm hitting an error attempting to use `apb relist` (and also `apb push`).

```
$ apb relist
Relist failure: Received non-200 status code while retrieving broker: ansible-service-broker
Response body:
404 page not found
```

Quote from ernelson on IRC:
```
<ernelson> dwhatley: relist probably needs to be updated to reflect the new namechange
<ernelson> dwhatley: it's 404ing because the /servicebroker path doesn't exist anymore
```

```
$ rpm -qa | grep apb
apb-1.0.0-1.20171009135441.fc25.noarch
```

**Comment by djwhatle** [comment]
Spoke more with @eriknelson and @shawn-hurley , sounds like this can be fixed by a single-line change.

https://github.com/ansibleplaybookbundle/ansible-playbook-bundle/blob/master/src/apb/engine.py#L370

Needs to be changed to:
`return "{}/apis/servicecatalog.k8s.io/v1beta1/clusterservicebrokers/{}".format(host, broker_name)`

Am going to submit a PR with this small change.

**Comment by eriknelson** [comment]
Fixed via #145
